defmt/tests/ui.rs

```diff
@@ -9,5 +9,6 @@ fn ui() {
         t.compile_fail("tests/ui/*.rs");
 
         t.pass("tests/basic_usage.rs");
+        t.pass("tests/derive-bounds.rs");
```

Comment:
Does this actually work? I don't see this test being executed with the other ui tests, but instead as a "normal" test

Reply:
I get this output when running `cargo test --features unstable-test`:
```
[TRUNCATED]
     Running tests\ui.rs (defmt\target\debug\deps\ui-a6effd36e167384c.exe)

running 1 test
   Compiling syn v2.0.41
   Compiling thiserror-impl v1.0.50
   Compiling thiserror v1.0.50
   Compiling defmt-parser v0.3.3 (defmt\parser)
   Compiling defmt-macros v0.3.6 (defmt\macros)
   Compiling defmt v0.3.5 (defmt\defmt)
   Compiling defmt-tests v0.0.0 (defmt\target\tests\trybuild\defmt)
    Finished dev [unoptimized + debuginfo] target(s) in 9.76s


test tests\ui\derive-empty-attr.rs [should fail to compile] ... ok
test tests\ui\derive-invalid-attr-arg.rs [should fail to compile] ... ok
test tests\ui\derive-multi-attr.rs [should fail to compile] ... ok
test tests\ui\log-invalid-hint.rs [should fail to compile] ... ok
test tests\ui\panic-handler-argument.rs [should fail to compile] ... ok
test tests\ui\panic-handler-empty.rs [should fail to compile] ... ok
test tests\ui\panic-handler-export-name.rs [should fail to compile] ... ok
test tests\ui\panic-handler-no-mangle.rs [should fail to compile] ... ok
test tests\ui\write-invalid-hint.rs [should fail to compile] ... ok
test tests/basic_usage.rs [should pass] ... ok
test tests/derive-bounds.rs [should pass] ... ok


test ui ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 15.43s
[TRUNCATED]
```

So I thought this is working. Did I miss something?

Reply:
It does work. Somehow I didn't see it